lib/industry_mapping_client.rb

```diff
@@ -7,7 +7,7 @@ class IndustryMappingClient
   BASE_URL = "http://#{HOST}/api/industries.json"
 
   def self.map_industry(topic, source)
-    results = HTTParty.get(BASE_URL, query: {topic: topic, source: source})
+    results = HTTParty.get(BASE_URL, query: {topic: topic, source: source, log_failed: true})
```

Comment:
I think we should do something like what's described in this [SO answer](http://stackoverflow.com/a/10112179):

Have a `configure` method on `IndustryMappingClient`, to which we can pass the `log_failed` option. It should be false by default, and we'll set it to true in production:

```
config/initializers/industry_mapping_client.rb:

if Rails.env.production?
  IndustryMappingClient.configure(log_failed: true)
end
```